src/main/java/shop/cazait/domain/cafe/api/CafeController.java

```diff
@@ -0,0 +1,39 @@
+package shop.cazait.domain.cafe.api;
+
+import lombok.RequiredArgsConstructor;
+import org.springframework.web.bind.annotation.*;
+import shop.cazait.domain.cafe.dto.GetCafeRes;
+import shop.cazait.domain.cafe.service.CafeService;
+import shop.cazait.global.common.response.BaseResponse;
+import shop.cazait.global.common.status.BaseStatus;
+
+import java.util.List;
+
+@RestController
+@RequestMapping("/api/cafes")
+@RequiredArgsConstructor
+public class CafeController {
+
+    private final CafeService cafeService;
+
+    @ResponseBody
+    @GetMapping("/all")
+    public BaseResponse<List<GetCafeRes>> getCafeByStatus() {
+        List<GetCafeRes> cafeResList = this.cafeService.getCafeList(BaseStatus.ACTIVE);
+        return new BaseResponse<>(cafeResList);
+    }
```

Comment:
`ResponseBody` 어노테이션은 불필요 합니다..!
이유는 `RestController`가 `ResponseBody`와 `Controller`를 합쳐서 만든 어노테이션 이기 때문입니다!

아래는 `RestCotroller` 구현 코드인데
```
@Target(ElementType.TYPE)
@Retention(RetentionPolicy.RUNTIME)
@Documented
@Controller
@ResponseBody
public @interface RestController {

	/**
	 * The value may indicate a suggestion for a logical component name,
	 * to be turned into a Spring bean in case of an autodetected component.
	 * @return the suggested component name, if any (or empty String otherwise)
	 * @since 4.0.1
	 */
	@AliasFor(annotation = Controller.class)
	String value() default "";

}
```
설명과 같이 `ResponseBody`가 이미 선언되어 있음을 알 수 있습니다.